Dockerfile

```diff
@@ -207,6 +229,86 @@ RUN mkdir src &&\
     cmake \
         --build src/build \
         --target install \
+#	-j$(nproc) \
     &&\
     ln -s ${__prefix}/bin/thisdd4hep.sh ${__ldmx_env_script_d__}/thisdd4hep.sh &&\
     rm -r src
+
+###############################################################################
+# LHAPDF
+#
+# Needed for GENIE
+#
+###############################################################################
+LABEL lhapdf.version="6.5.2"
+ENV PYTHON_VERSION=3
+RUN mkdir src &&\
+    ${__wget} https://lhapdf.hepforge.org/downloads/?f=LHAPDF-6.5.2.tar.gz |\
+     ${__untar} &&\
+    cd src &&\
+    ./configure --prefix=${__prefix} &&\
+    make -j$(nproc) &&\
+    make -j$(nproc) install &&\
+    cd ../ &&\
+    rm -rf src
+
+###############################################################################
+# Installing log4cpp
+#
+# Needed for GENIE
+#
+###############################################################################
+
+RUN apt-get update &&\
+    apt-get install -y autoconf automake libtool
```

Comment:
Move these to the `apt-get` machinery above.

Same comments about putting them in alphabetical order and inserting them into the docs.

Reply:
Done.